.github/workflows/ci.yml

```diff
@@ -15,10 +15,12 @@ jobs:
         with:
           distribution: 'adopt-hotspot'
           java-version: '11'
-          cache: 'maven'
+      - uses: gradle/gradle-build-action@v2
+        with:
+          gradle-version: 7.4.1
```

Comment:
Is it possible to manage these versions at a central place to avoid having to update more than one place, e.g. a new Gradle version is released?

Reply:
The alternative is using the wrapper, but this will mean we'd be downloading and extracting the wrapper every time we run the CI. I don't think it's critical to have this in multiple places. If it becomes a chore we can still put a helper script into the repo that updates all occasions.